spec/tags/truffle/launcher_tags.txt

```diff
@@ -13,3 +13,11 @@ slow:The launcher prioritises options on the command line over system properties
 slow:The launcher prioritises options on the command line using -X over system properties
 slow:The launcher allows -cp in JAVA_OPTS
 slow:The launcher allows -classpath in JAVA_OPTS
+aot:The launcher prints the full java command with -J-cmd
+aot:The launcher prints the full java command with -cmd in JAVA_OPTS
+aot:The launcher adds options from $JAVA_OPTS to the command
+aot:The launcher preserve spaces in options
```

Comment:
I'm going to take another pass over these to see if we can do something better for AOT. Right now I just want to get "jt test spec" running 100% so I can add to CI.